isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -1323,6 +1428,18 @@ namespace isobus
 		return retVal;
 	}
 
+	bool VirtualTerminalClient::get_vt_version_supported(VTVersion minimumVersion)
+	{
+		bool retVal = false;
+
+		if (get_connected_vt_version() != VTVersion::ReservedOrUnknown)
+		{
+			retVal = get_connected_vt_version() >= minimumVersion;
```

Comment:
`get_connected_vt_version` is `const` so this function can probably also be `const`. 

`bool VirtualTerminalClient::get_vt_version_supported(VTVersion minimumVersion) const` here and in the header file.